src/marqo/tensor_search/backend.py

```diff
@@ -47,12 +47,21 @@ def get_index_info(
         raise errors.NonTensorIndexError(
             f"Error retrieving index info for index {index_name}")
 
+    # Identify index `model` from mapping metadata
     if "model" in res[index_name]["mappings"]["_meta"]:
         model_name = res[index_name]["mappings"]["_meta"]["model"]
     else:
         raise errors.NonTensorIndexError(
             "get_index_info: couldn't identify embedding model name "
             F"in index mappings! Mapping: {res}")
+    
+    # Identify index `search_model` from mapping metadata
+    if "search_model" in res[index_name]["mappings"]["_meta"]:
```

Comment:
This doesn't appear to be backwards compatible for existing indexes?

Reply:
fixed, setting `search_model_name` to `None` if it doesn't appear in OS response.